src/PatchKit.Api/swagger-main.json

```diff
@@ -76,7 +168,23 @@
             "schema": {
               "type": "array",
               "items": {
-                "$ref": "#/definitions/Changelog"
```

Comment:
Generator doesn't support `"type" : "object"` and because of that we need to define each of structure at the end of the swagger document. I see that it didn't make any harm in this pull request but it's important to remember that swagger document needs to be fixed.